src/components/SlateEditor/EditorFooter.tsx

```diff
@@ -28,12 +29,13 @@ interface Props {
   values: Values;
   error: string;
   errors: Object;
-  getArticle: () => Article;
-  articleStatus: { current: string };
+  getEntity: () => Article | Concept;
+  entityStatus: { current: string };
   createMessage: (o: { translationKey: string; severity: string }) => void;
   showSimpleFooter: boolean;
   setFieldValue: (name: string, value: { current: string }) => void;
   onSaveClick: VoidFunction;
+  entityType: string;
```

Comment:
Om du sender inn getStateMachine som callback kan denne endres til for eksempel isArticle og så kan du bruke den for å vise validering og forhåndsvisningsfunksjonalitet.